assets/api.ts

```diff
@@ -0,0 +1,24 @@
+import {IArticle} from './interfaces';
+import {gettext} from './utils';
+
+export interface INewshubApi {
```

Comment:
maybe we should call it like `INewshubExtensionApi` if we expect it to be overriden, I would expect `INewshubApi` something I can use but not change